modules/git/commit.go

```diff
@@ -177,13 +190,18 @@ func CommitsCountFiles(ctx context.Context, repoPath string, revision, relpath [
 }
 
 // CommitsCount returns number of total commits of until given revision.
-func CommitsCount(ctx context.Context, repoPath string, revision ...string) (int64, error) {
-	return CommitsCountFiles(ctx, repoPath, revision, []string{})
+func CommitsCount(ctx context.Context, opts CommitsCountOptions) (int64, error) {
+	return CommitsCountFiles(ctx, opts)
```

Comment:
Looks like the two functions now could be merged as one?

Reply:
Yep, I'll get rid of `CommitsCountFiles` and just use `CommitsCount` everywhere